libretroshare/src/gxs/rsgxsnetservice.cc

```diff
@@ -877,7 +879,7 @@ void RsGxsNetService::syncGrpStatistics()
 			for(uint32_t i=0;i<n;++i)
 				++rit ;
 
-			for(uint32_t i=0;i<std::min(rec.suppliers.ids.size(),(size_t)GROUP_STATS_UPDATE_NB_PEERS);++i)
+			for(size_t i=0;i<std::min(rec.suppliers.ids.size(),GROUP_STATS_UPDATE_NB_PEERS);++i)
```

Comment:
I prefer the previous version: .size() is a size_t, so the int GROUP_STATS... should be cast to size_t. So in summary: keep GROUP_STATS_... to be an int. Keep the size_t in the for()

Reply:
GROUP_STATS_UPDATE_NB_PEERS is only used here.